pkg/rewards/claim.go

```diff
@@ -154,13 +237,59 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 	logger.Info("Validating claim proof...")
 	ok, err := elReader.CheckClaim(ctx, elClaim)
 	if err != nil {
-		return err
+		return nil, nil, nil, err
 	}
 	if !ok {
-		return errors.New("failed to validate claim")
+		return nil, nil, nil, errors.New("failed to validate claim")
 	}
 	logger.Info("Claim proof validated successfully")
 
+	return &elClaim, claim, accounts, nil
+}
+
+func Claim(cCtx *cli.Context, p utils.Prompter) error {
+	ctx := cCtx.Context
+	logger := common.GetLogger(cCtx)
+
+	config, err := readAndValidateClaimConfig(cCtx, logger)
```

Comment:
we don't need to `readAndValidateClaimConfig` twice - we can just read once and pass through